schemes/utilities/to_be_ccppized_temporary.F90

```diff
@@ -0,0 +1,29 @@
+module to_be_ccppized_temporary
```

Comment:
Perhaps a call to this has to be added to the CAM7 SDF as well which will use ZM?

Reply:
@nusbaume and @peverwhee  - What are your thoughts on adding this to the CAM7 SDF?  By the time the CAM7 SDF is complete, I think a line like this might be lost in SDF.   I'm not sure I'd like to add this to the SDF for this reason, but on the other hand, we might need it as a stopgap measure to get CAM7 running initially and we will most likely have forgotten about it?

Reply:
I guess I'll vote for leaving it in. My hope is we will delete `to_be_ccppized_temporary.F90` when it's time, which will preclude us from accidentally leaving it in the SDF

Reply:
As I thought about how to implement this in CAM7, I believe it already is inherently in the CAM7 SDF, though it is not called out by name.  I believe it was needed for the subsequent wv_sat_run which is used internally in one of the ZM routines.  For the reason why I needed to add this routine, I believe that the current ZM is calling this, though not explicitly in the SDF.  So I don't believe I need to do anything more, but please let me know if I misunderstood the request.

Reply:
If I remember correctly, you added the `to_be_ccppized_temporary` scheme to call `wv_sat_init`. Is that being called by a different scheme in the SDF? If not, we'll need it in the SDF or the call to `wv_sat_run` that's internal to ZM will fail.

Reply:
Well, it turns out that it is at the top of both the `suite_zhang_mcfarlane.xml` as well as the top of the ZM section in the CAM7 suite.  So it is being tested - I just missed seeing it in my scanning of the files.